public/index.html

```diff
@@ -21,42 +21,33 @@
             <div class="col">
                 <h3> Welcome to the RERUM Geolocator </h3>
                 <p class="intro">
-                    Here you can enhance any <a target="_blank" href="https://www.w3.org/standards/semanticweb/data.html">Linked Open Data</a> resource by providing geospatial descriptors. This does not affect the original resource nor does it notify the resource owner.  There is no user system and so data generated here cannot be attributed to you specifically.  However, it will be available to you to make it your own by taking it through applications of your choosing.
+                    Here you can enhance any <a target="_blank" href="https://www.w3.org/standards/semanticweb/data.html">Linked Open Data</a> resource by providing geospatial descriptors. This does not affect the original resource nor does it notify the resource owner. You are able to attribute your data to yourself, as well as download your object as a JSON file at the end.
                 </p>
             </div>
         </div>
-        <div class="row">
-            <div class="col">
-                <h4> <abbr title="International Image Interoperability Framework">IIIF</abbr> Resources </h4>
-                <p>
-                    This application is proud to support the <a target="_blank" href="https://iiif.io/api/presentation/">IIIF Presentation API</a> and encourages you to bring your own IIIF resources.  If your web resource fits this category, there are two main paths to take.
-                </p>
-                <h5> navPlace </h5>
+
+        <div class="row" >
+            <div class="col text-center" style="padding-right:20px; padding-left:10px; background: radial-gradient(circle, #bdd5bd, #e4ece4); border-radius: 15px;">
+                <h4> Create a navPlace object </h4>
                 <p>
```

Comment:
This `<p>` doesn't close!  When you close it, you can give it a padding or margin to get the spacing effect you are going for (this is in relation to the `br` comment).